packages/pdk-nag/src/packs/aws-prototyping-rules.ts

```diff
@@ -13,25 +13,990 @@ export let PackName = "AwsPrototyping";
+    level: NagMessageLevel.ERROR,
+    rule: rules.autoscaling.AutoScalingLaunchConfigPublicIpDisabled,
+  },
+  {
+    info: "The Cloud9 instance does not use a no-ingress EC2 instance with AWS Systems Manager.",
+    explanation:
+      "SSM adds an additional layer of protection as it allows operators to control access through IAM permissions and does not require opening inbound ports.\n\nExample threat: A global internet-based actor  can discover Cloud9 EC2 instances that have public IP addresses and that are exposing SSH, which may lead to reconnaissance and intrusion activities being performed against the exposed attack surface possibly impacting the confidentiality, integrity and availability of the data and resource assets associated with the prototype",
+    level: NagMessageLevel.ERROR,
+    rule: rules.cloud9.Cloud9InstanceNoIngressSystemsManager,
+  },
+  {
+    info: "The CloudFront distribution may require Geo restrictions.",
+    explanation:
+      "Geo restriction may need to be enabled for the distribution in order to allow or deny a country in order to allow or restrict users in specific locations from accessing content.\n\nExample threat: A global internet-based actor  can discover prototype web assets that are exposed via CloudFront distributions, which may lead to recon and intrusion activities being performed against the exposed attack surface possibly impacting the confidentiality, integrity and availability of the data and resource assets associated with the prototype",
+    level: NagMessageLevel.ERROR,
+    rule: rules.cloudfront.CloudFrontDistributionGeoRestrictions,
+  },
+  {
+    info: "The CloudFront distribution allows for SSLv3 or TLSv1 for HTTPS viewer connections.",
+    explanation:
+      "Vulnerabilities have been and continue to be discovered in the deprecated SSL and TLS protocols. Help protect viewer connections by specifying a viewer certificate that enforces a minimum of TLSv1.1 or TLSv1.2 in the security policy. Distributions that use that use the default CloudFront viewer certificate or use 'vip' for the SslSupportMethod are non-compliant with this rule, as the minimum security policy is set to TLSv1 regardless of the specified MinimumProtocolVersion",
+    level: NagMessageLevel.WARN,
+    rule: rules.cloudfront.CloudFrontDistributionHttpsViewerNoOutdatedSSL,
+  },
+  {
+    info: "The CloudFront distributions uses SSLv3 or TLSv1 for communication to the origin.",
+    explanation:
+      "Vulnerabilities have been and continue to be discovered in the deprecated SSL and TLS protocols. Using a security policy with minimum TLSv1.1 or TLSv1.2 and appropriate security ciphers for HTTPS helps protect viewer connections.",
+    level: NagMessageLevel.WARN,
+    rule: rules.cloudfront.CloudFrontDistributionNoOutdatedSSL,
+  },
+  {
+    info: "The CloudFront distribution does not use an origin access identity an S3 origin.",
```

Comment:
Ack on double spacing will fix.

Agree on OAC, however this would require either a change to the existing cdk-nag rule, or be a new rule to be written. Will add to backlog